src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/hubandspoke/WorkerBolt.java

```diff
@@ -87,7 +87,8 @@ private void dispatchResponse(Tuple input) throws Exception {
         if (request != null) {
             onAsyncResponse(request, input);
         } else {
-            unhandledInput(input);
+            log.error("{} drop worker async response. because {} key is not listed in pending response list",
```

Comment:
maybe it's worth to log tuple payload as well? also, have you checked it doesn't spam a ton of errors?